content/altinn-studio/getting-started/app-dev-course/modul2/_index.nb.md

```diff
@@ -35,36 +35,34 @@ Datamodellen definerer hvilke data man forventer å samle inn, og hvilket format
 For et enkelt skjema er det ofte en 1-1 knytning fra felter i datamodellen til felter i skjemaet, mens for mer avanserte
```

Comment:
For et enkelt skjema er det ofte en én-til-én-knytning mellom felter i datamodellen og felter i skjemaet. For mer avanserte skjemaer kan datamodellen og skjemaet være ganske forskjellige.